pages/supportus.vue

```diff
@@ -91,6 +91,17 @@
         </div>
       </div>
     </div>
+    <div>
+      <section-title path="supportus.incometax.title" />
+      <paragraph>
+        {{ $t('supportus.incometax.p1') }}
+        <i18n path="supportus.incometax.p2.content" tag="p" class="paragraph-sm xl:paragraph">
+          <nuxt-link class="text-link" :to="localePath('/incometax')">
+            {{ $t("supportus.incometax.p2.redirect_label") }}
+          </nuxt-link>
+        </i18n>
+      </paragraph>
```

Comment:
I think it was not your intention here to have a paragraph inside another paragraph.
So closing the first paragraph here will solve our problem.
```suggestion
      <paragraph path="supportus.incometax.p1" :margin-bottom="false"/>
      <i18n path="supportus.incometax.p2.content" tag="p" class="paragraph-sm xl:paragraph">
          <nuxt-link class="text-link" :to="localePath('/incometax')">
            {{ $t("supportus.incometax.p2.redirect_label") }}
          </nuxt-link>
      </i18n>
```

Otherwise we end up with an **mb-4** from the first paragraph and a `<p><p>...` structure.
<img width="948" alt="supportus_redirect_too_much_spave" src="https://user-images.githubusercontent.com/1261457/104599264-36324980-5680-11eb-88af-7edd78b2a4a4.PNG">

Alternatively you could go for `incometax` as an array approach and use the `paragraphs` component but I think your approach is better here as we only have two paragraphs.

Reply:
Done